Automattic/vip-design-system
# Converting Code component to TypeScript

## Description

This pull request coverts the Code component to TypeScript.

## Checklist


## Steps to Test


1. Pull down PR.
1. `npm run dev`.
1. Open Storybook.
1. Check the accordion is working as expected.
